route/rdrp.py

```diff
@@ -35,3 +35,58 @@ def get_rdrp_counts_route():
 def get_rdrp_list_route(query_type):
     values_list = rdrp_query.get_list(query_type, **request.args)
     return jsonify(values_list)
+
+@app.route('/pos/rdrp')
+def get_rdrp_pos_route():
+    """
+    Function to return the rdrp_pos table data from the SQL server
+
+    Returns:
+        A JSON response containing the result of the query in the form of a list of dictionaries. If the page or perPage
+        parameters are invalid, an error message is returned with a status code of 400.
+    """
+    if not request.args.get('page'):
+        page = 1
+    else:
+        page = validate_args(request.args['page'])
+    if not page:
+        error = {'message': 'Invalid page parameter: {arg}'.format(arg = request.args['page'])}
+        return make_response(jsonify(error), 400)
+
+    if not request.args.get('perPage'):
+        perPage = 20
+    else:
+        perPage = validate_args(request.args['perPage'])
+    if not perPage:
+        error = {'message': 'Invalid perPage parameter: {arg}'.format(arg = request.args['perPage'])}
+        return make_response(jsonify(error), 400)
```

Comment:
Pagination error handling and default values are already handled by `flask-sqlalchemy` so it seems like this validation and the helper can be removed ([see docs](https://flask-sqlalchemy.palletsprojects.com/en/2.x/api/#flask_sqlalchemy.BaseQuery.paginate)).